pages/content/posts/complete-interop-features/index.md

```diff
@@ -0,0 +1,73 @@
+
+Since the early 2010s (and even prior), standard bodies and browser makers have rapidly specified and implemented new APIs. MDN's goal was to document these new features. As these features were web developers' main focus, extensive, accurate, and current documentation was (and continues to be) required. The originally named "HTML5", "CSS3", and "ES6" initiatives allowed MDN to revamp these areas of documentation because there was an interest in positioning the web as a recognized interoperable and modern platform.
+
+
+![The three logos of HTML5, CSS3, and JS](logos.png)
+
+The development of the web platform at a greater speed led to the MDN writing community being unable to keep up with documenting all new features while also trying to document the existing foundations of the web platform. This sometimes led to mitigating actions, like listing methods and properties on the interface page but not creating the sub-pages.
+
+Today's gaps mostly fall into three categories:
+
+- HTML DOM and SVG DOM features added to the web platform when MDN was still in its infancy;
+- non-interoperable features: APIs supported by only one or two browser engines;
+- single properties and methods that were forgotten when the interface was initially documented or added later.
+
+## Opportunities created by the Interop initiative
+
+As a collective of writers dedicated to documenting the open web platform, [Open Web Docs (OWD)](https://openwebdocs.org/) is able to _close these gaps_.  We believe that projects like this can significantly improve in the middle- and long-term health and the longevity of the documentation. We aim to provide complete documentation of the web platform.
+
+For several years now, browser makers have come together to discuss web developer pain points, defining the features that need more attention to improve cross-browser functionality and fixing inconsistencies and issues, ensuring web devs can implement features reliably. This is known as the [Interop](https://web.dev/interop-2023/) initiative.
+
+OWD's Steering Committee has determined these projects provide excellent opportunities to revisit the relevant areas of MDN Web Docs documentation. The objective and goal is to have complete documentation for each set of features made interoperable over the year. 
```

Comment:
nit: maybe link to the steering committe page on OWD's website.

Reply:
Great idea!